tests/integ/test_query_history.py

```diff
@@ -195,7 +194,7 @@ def test_query_history_with_multi_thread_and_describe(session):
             _, _ = wait(works, return_when=ALL_COMPLETED)
     thread_numbers = set()
     for query in query_history.queries:
-        assert query.sql_text.split(" ")[-1] == str(query.thread_id)
+        if not getattr(query, "is_describe", False):
+            assert query.sql_text.split(" ")[-1] == str(query.thread_id)
```

Comment:
do we have to use `getattr`? We are creating query history object with `include_describe=True`

Reply:
we need to use it. we will have two type of record in query history, one is the actual query, which does not have `include_describe `, the other is describe query, which have `include_describe`